### ISSUE TITLE
I have a question regarding GEM-2 model, PCQM4Mv2 dataset

### ISSUE THREAD

**Issue by Sangyeup** [PaddlePaddle/PaddleHelix#243]
Hi, I have a question regarding GEM-2 model.

How did you measure the performance of validation set and test set on PCQM4Mv2 dataset?

Because I knew that they do not provide 3D coordinates.

Thank you.

**Comment by Noisyntrain** [comment]
Hi Sangyeup, yes we have measured the proformance of validation and test sets on PCQM4Mv2, you reference to the readme: https://github.com/PaddlePaddle/PaddleHelix/tree/dev/apps/pretrained_compound/ChemRL/GEM-2. As for the 3D coordinates, we didn't use the provided 3D coordinates, instead, we used the corrdinates calculated by the RDKIT to train and test models' performance on PCQM4Mv2.

**Comment by Sangyeup** [comment]
I see. Thanks for the answer.

Then I have one more question, have you tried training with provided 3D coordinates on training set and validating with 3D coordinates calculated by the RDKIT on test set?

I wonder which one will produce better results.

**Comment by Noisyntrain** [comment]
We didn't try this training method. Maybe you can try it and compre the reuslts.
